README.md

```diff
@@ -63,16 +63,14 @@ client = RubyAmazonBedrock::Client.new(
 )
 ```
 
-## With Configuration
+### AWS Named Profiles
 
-```ruby
-RubyAmazonBedrock.configure do |config|
-  config.region = ENV.fetch('AWS_REGION', nil)
-  config.access_key_id = ENV.fetch('AWS_ACCESS_KEY_ID', nil)
-  config.secret_access_key = ENV.fetch('AWS_SECRET_ACCESS_KEY', nil)
-end
+You can also use [AWS Named Profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-format-profile) by passing the `profile` keywoard argument. When using a named profile, specyfing the `region`, `access_key_id` and `access_token` won't be required.
```

Comment:
Don't remove the With Configuration documentation, you can add the AWS Named Profiles note below that.